src/Components/avatarUser/AvatarUser.js

```diff
@@ -0,0 +1,39 @@
+import { Avatar } from "@mui/material";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import { useState, useEffect } from "react";
+
+import Modal from "../modal/Modal";
+import { paintAvatarAndName } from "../../utils";
+import "./avatarStyles.scss";
+
+const AvatarUser = () => {
+  const [seeModal, setSeeModal] = useState(false);
+  const [textAvatar, setTextAvatar] = useState("");
+  const [userName, setUserName] = useState("");
+
+  useEffect(() => {
+    paintAvatarAndName(setTextAvatar, setUserName);
```

Comment:
lets rename this function to `drawAvatarAndName`

Reply:
I am going to use this function getAvatarAndName() to fetch data from local storage